static/styles/portico/portico-signin.scss

```diff
@@ -832,15 +832,28 @@ button#register_auth_button_gitlab {
         }
     }
 
-    .help-box {
-        width: calc(100% - 25px);
+    #id_email {
+        display: inline-block;
+        font-weight: normal;
+        margin: 25px 2px 2px;
+        text-align: left;
+        width: 100%;
+        overflow-wrap: break-word;
+    }
+
+    .help-text {
+        width: 100%;
         max-width: none;
-        margin: 0 auto;
+        margin: 2px 0;
+        text-align: left;
+        color: hsl(0, 0%, 47%);
+        font-size: 0.9rem;
+        font-weight: 400;
+        line-height: 1.2;
+    }
 
-        &.margin-top {
-            margin-top: 21px;
-            text-align: left;
-        }
+    legend {
+        margin-bottom: -5px;
```

Comment:
I consider negative margins and paddings to be generally bad style (yes I know the Zulip CSS uses them occasionally). I think we could probably achieve the same by reducing some positive margin or padding.

Reply:
Fixed this.